### DISCUSSION TITLE
Works and doesn't work at the same time.

### DISCUSSION THREAD

**Discussion by JamesFraserLogan** [community/community#81199] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Be my repo. Be able to push/pull from the repo. But not clone. On the same machine.

Be my other repo. Have no access rights while I can still push/pull my other repo.

I don't know what changed today, but I need to clone my work and even though I can push/pull to my other repo (both private, owned by me), I can't clone.

I really detest bugs like this. I should either be able to not push/pull and not clone or be able to both push/pull and clone.

There are no perms issues, it works, I don't have a key on this flavour of this operating system (don't ask, i have lots of profiles; cygwin, wsl etc) but I could pull it the other day without one?

This makes no sense.

**Comment by ananuness** [comment]
This is really confusing for sure hehe but can you check this step by step: [Troubleshooting cloning errors](https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors).

And maybe you can fix it with this one [Managing remote repositories](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories). Checking if the url is correct and changing if needed.
